For DG Draft. To add in code for find appt also.

tanweili opened the pull request.

tanweili committed:
Add basic text for ApptList.find() feature

For DG draft deadline. To add in puml and update code to implement
find appointment features.

wraineflores commented on docs/DeveloperGuide.md:
```diff
@@ -93,14 +93,28 @@ How the Command class works:
 ###Target user profile:
 
-* has a need to manage a significant number of contacts
+* lightweight appli
```
application/app*

wraineflores reviewed (approved):
Looks good since it's similar to the sample DG!

tanweili committed:
Add find appointment functionality

Add code across Manager, Command, Parser, Validator in their
respective standard to have find appointment in the front end.

tanweili commented:
PUML diagram shall be added in future PR.
